config/full-stack.json

```diff
@@ -0,0 +1,304 @@
+{
+  "account": {
+    "test": {
+      "password": "098f6bcd4621d373cade4e832627b4f6"
+    }
+  },
+  "autopushes": [
+    [
+      "videorec+",
+      "s3+http://admin:password@localhost:9000/os-recordings/$wildcard/$uuid/source/$segmentCounter.ts?m3u8=../output.m3u8&split=5&video=source&audio=AAC&append=1&waittrackcount=2&recstart=-1"
+    ]
+  ],
+  "config": {
+    "accesslog": "LOG",
+    "controller": {
+      "interface": null,
+      "port": null,
+      "username": null
+    },
+    "defaultStream": null,
+    "limits": null,
+    "prometheus": "koekjes",
+    "protocols": [
+      {
+        "connector": "livepeer-cockroach"
+      },
+      {
+        "connector": "livepeer-rabbitmq"
+      },
+      {
+        "connector": "livepeer-nginx"
+      },
+      {
+        "connector": "livepeer-minio"
+      },
+      {
+        "connector": "livepeer-core-dump-monitor"
+      },
```

Comment:
I'd find systemd services more reliable mechanism to manage db, nginx, rabbitmq... 
Also long-term they might need to be separate docker-compose services so we wouldn't need to restart them and wait for boot when rebuilding `in-a-box` container.

Reply:
We chatted about this, but for the record: yes, we're using MistController in kind of a silly way here, but the eventual goal is to make this all a single statically-linked binary by removing and merging external dependencies. So I don't really want to go too far down the systemd path, as that seems like we're getting farther away from that reality.